### DISCUSSION TITLE
Commodore CVR 200 - blinking after flash

### DISCUSSION THREAD

**Discussion by Crunch91** [Hypfer/Valetudo#2009] [Q&A / Support]
I flashed my Commodore CVR 200 as described here:
https://github.com/Hypfer/valetudo-crl200s-root

After finishing the guide I rebooted the robot and since then both buttons (home and power) are blinking slowly in white color. Neither I can reboot it anymore nore I can reset Wifi.

NOTE: Step 6 from the guide above I could not execute because no such file is existing on my robot:

GUIDE:

>So far, we've identified a few models that have the same hardware but with a different Wi-Fi module (RTL8821CS instead of the RTL8189ES):

>Commodore CVR 200

>Simply open a shell with adb shell and execute /opt/8821cs/enable_8821cs.sh. Then, reboot and you're done

There is no folder named `/opt` on my Commodore CVR 200.

But I don't know if this has anything to do with the blinking.
Before I try to restore it with the backup files I want to know if there is anything I can try to make it run?

**Comment by Hypfer** [comment]
> After finishing the guide I rebooted the robot and since then both buttons (home and power) are blinking slowly in white color. Neither I can reboot it anymore nor I can reset Wifi.

Sounds like you might have re-ran the enable-adb.sh and with that disabled the core robot process again.
That's a somewhat common mistake to make.
Alternatively, and since you say that there's no opt on there, you might've not flashed the rooted firmware.


Make sure to thoroughly read the rooting guide and follow it carefully step by step.

**Reply by Crunch91** [reply]
thanks for your reply.

I didn't even download `enable-adb.sh`.

Just tried to connect with adb but this doesnt work anymore. According to the guide I did a backup but without adb connection I can't use it.

I used following image:
https://builder.dontvacuum.me/_viomi_v6.html

The guide was quite straight ahead. I just reviewed the steps I did:

1. `chmod +x that file and then adb push ./adb_shell /bin/adb_shell.`
2. (pull backup)
3. `adb push viomi.vacuum.v6_fw.tar.gz.tar.gz /tmp/`
4. `adb push ./valetudo-armv7.upx /mnt/UDISK/valetudo.`
5. `adb push ./convert-robot.sh /tmp/`
6. `adb shell`
`cd /tmp/`
`sh ./convert-robot.sh`
`tar xzvf ./<rooted_firmware_filename>.tar.gz`
`sh ./install.sh`

The guide says:
> If you're only seeing the adb device show for a fraction of a second on boot

But where should I see that? (I use knoppix)

**Reply by Hypfer** [reply]
Sorry, but GitHub is not the right place for these kinds of support ping-pong walkthroughs.
